desktop/desktop.scss

```diff
@@ -51,26 +51,70 @@
 }
 
 // Recent replies
-.recent-replies--reply {
-  border-bottom: 1px solid var(--primary-low);
-  padding-bottom: 0.5em;
-  margin-bottom: 1em;
-  display: flex;
-}
+.recent-replies {
+  &--reply {
```

Comment:
Just a code styling not, personally, I am not a fan of splitting an element's class name like this. Makes it harder to find in the future.